ci/Jenkinsfile

```diff
@@ -15,10 +15,8 @@ pipeline {
     PYTHONPATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/lib/python:${PYTHONPATH}"
     LD_LIBRARY_PATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/python3/lib:${LD_LIBRARY_PATH}"
 
-    TESTRAIL_CREDS = credentials('test-rail-api-devops')
     TESTRAIL_URL = "https://ethstatus.testrail.net"
-
-    CI_CREDS = credentials('status-im-auto')
+    TESTRAIL = credentials('test-rail-api-devops')
```

Comment:
please don't set credentials in global `environment` section. Use `withCredentials` in the stage in which it is needed.

Reply:
fixed